src/Symfony/Bridge/PhpUnit/Tests/ClockMockTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\ClockMock;
+
+class ClockMockTest extends \PHPUnit_Framework_TestCase
+{
+    public function testTimeNowNull()
+    {
+        $this->assertTrue(is_int(ClockMock::time()));
+    }
+
+    public function testSleepNowNull()
+    {
+        $this->assertEquals(0, ClockMock::sleep(0));
+    }
+
+    public function testUsleepNowNull()
+    {
+        $this->assertNull(ClockMock::usleep(0));
+    }
+
+    public function testMicrotimeNowNull()
+    {
+        $this->assertTrue(is_float(ClockMock::microtime(true)));
+    }
+
+    public function testWithClockMock()
+    {
+        $this->assertFalse(ClockMock::withClockMock());
```

Comment:
ok, with no argument there is no side effect, so the above comment applies to line 45 below